docs/_docs/installation/macos.md

```diff
@@ -3,155 +3,28 @@ title: Jekyll on macOS
 permalink: /docs/installation/macos/
 ---
 
-## Install Command Line Tools
-To install the command line tools to compile native extensions, open a terminal and run:
+## Supported macOS versions
 
-```sh
-xcode-select --install
-```
+- Monterey (macOS 12)
+- Big Sur (macOS 11)
+- Catalina (macOS 10.15)
 
-### set SDKROOT (only macOS Catalina or later)
```

Comment:
In the past ten years of installing Ruby on a Mac on every OS X and macOS release, I have never had to set `SDKROOT`. This is absolutely not needed.